src/App.js

```diff
@@ -9,7 +9,9 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData,
+      movies: {
```

Comment:
Nice work. I'll also suggest that we could've done `movies: []` and in the fetch call access the movies.